<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="ModeComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="FindAndConnectButton">
      <property name="text">
       <string>Find and Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StartVideoButton">
      <property name="text">
       <string>Start Video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopVideoButton">
      <property name="text">
       <string>Stop Video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DisconnectButton">
      <property name="text">
       <string>Disconnect Camera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="VideoFeedLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
